justfile

```diff
@@ -185,8 +185,9 @@ new-security-rule name:
     cargo run -p rulegen {{name}} security
```

Comment:
The indentation on 173 - 185 lines are one level deeper than it should be, but I'm leaving it as is in this PR.
That's why the indentation doesn't match the lines above.